File: php/linearSearch.php
<?php
// 線形探索
function linear_Search($arr, $target) {
  for ($i = 0, $len = count($arr); $i <= $len; $i++) {
    if ($arr[$i] == $target) {
      return $i;
    }
  }
  return false;
}

echo linear_Search([104, 865, 413, 541, 304, 354, 165, 150], 304);

File: swift/linearSearch.swift
// 線形探索
func linearSearch(arr: [Int], target: Int) -> Int {
    for i in 0 ..< arr.count {
        if arr[i] == target {
            return i;
        }
    }
    return -1;
}

print(linearSearch(arr: [104, 865, 413, 541, 304, 354, 165, 150], target: 304));
